<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vec3PropertyWidget</class>
 <widget class="QWidget" name="Vec3PropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QDoubleSpinBox" name="xSpin">
     <property name="minimum">
      <double>-9999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="ySpin">
     <property name="minimum">
      <double>-9999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="zSpin">
     <property name="minimum">
      <double>-9999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
